### PR TITLE
ENH: Adding Displacement field python example

### PR DESCRIPTION
**Pull request by PranjalSahu**

### CODE REVIEW

File: examples/test_tsd_displacement.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
+# [THINSHELLDEMONS] Read Mesh and Create TSD object
```
Comment (dzenanz) [inline examples/test_tsd_displacement.py]: I think you need the [license header](https://github.com/InsightSoftwareConsortium/ITKMontage/blob/v0.7.3/examples/SimpleMontage.py#L1-L17) here.

File: examples/test_tsd_displacement.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+# Build the Module with Python Wrapping and import the itk package which is available under Wrapping/Generators/Python
+#import sys
+#sys.path.append('/home/pranjal.sahu/ITKPR7/ITK/ITK-build/Wrapping/Generators/Python')
```
Comment (dzenanz) [inline examples/test_tsd_displacement.py]: remove debug code

**Review by dzenanz — COMMENTED** [review]
mostly looks good on a glance

File: README.md [hunk 3]
```diff
@@ -11,16 +11,14 @@ This module implements the Thin Shell Demons regularization proposed in
...
-   capabilities of the ITK v4 registration framework.
-2. (TSHD) An implementation started by Qingyu Zhao and updated to fix some remaining task
-   and run with ITK 5
 
 > :warning: **This module requires to be compiled against an ITK version with**  
 > - [PointSetToPointSetMetricWithIndexv4](https://github.com/InsightSoftwareConsortium/ITK/pull/2385)   
 > 
-> If you are compiling against an older version  use the ones inlcuded in this repository marked with h/hxx-unused (remove the -unused).
+
+<p align="center">
```
Comment (dzenanz) [inline README.md:19]: Do we need HTML formatting? Why not put this image into the repository, and reference it using normal markdown syntax?

**Review by dzenanz — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (PranjalSahu) [reply README.md:19]: I needed it to change the size of image and to center it. I could not get such control using markdown syntax.

Comment (dzenanz) [reply README.md:19]: OK. Good to merge.

**Review by thewtex — APPROVED** [review]
Love it!

File: README.md [hunk 4]
```diff
@@ -11,16 +11,14 @@ This module implements the Thin Shell Demons regularization proposed in
...
+<img src="https://user-images.githubusercontent.com/1044135/158479969-7313ed94-c5fb-4803-ae8d-2c0631893664.png" width="400" height="300">
```
Comment (jcfr) [inline README.md:20]: Would it be possible to include a description for the image, you could something like this : 

|  |
|--|
| Left: This is ..., Right: This is ...|

```
| <img src="https://user-images.githubusercontent.com/1044135/158479969-7313ed94-c5fb-4803-ae8d-2c0631893664.png" width="400" height="300"> |
|--|
| Left: This is ..., Right: This is ...|
```

Comment (PranjalSahu) [reply README.md:20]: Yes I will add it. I am also getting results on the mouse dataset and will add them here too.

**Pull request merged**